docs/reference/data/widgets_by_platform.csv

```diff
@@ -1,4 +1,4 @@
-Component,Type,Component,Description,macOS,GTK+,Windows,iOS,Android,Django
+Component,Type,Component,Description,macOS,GTK+,Windows,iOS,Android,Web
 Application,Core Component,:class:`~toga.app.App`,The application itself,|y|,|y|,|y|,|y|,|y|,|y|
 Window,Core Component,:class:`~toga.window.Window`,Window object,|y|,|y|,|y|,|y|,|y|,|y|
 MainWindow,Core Component,:class:`~toga.app.MainWindow`,Main window of the application,|y|,|y|,|y|,|y|,|y|,|y|
```

Comment:
These tables are very inconvenient to use because you have to scroll them left and right all the time:

![widgets-by-platform](https://user-images.githubusercontent.com/166954/205443401-87dac052-3328-4b7b-8a46-4577d679eac8.png)

The only solution I can think of is to drop the Description column. If the user wants a summary of descriptions, they can see it on the reference/api/index page.

Reply:
I'd rather not have this table at all; it's only here because the inconsistent availability of widgets led to people asking for this kind of summary. Hopefully it won't be needed in the (not too distant) future; but in the meantime, I agree dropping the description column is the best option.